styles/modules/modals/_settings.scss

```diff
@@ -14,14 +14,21 @@
     }
   }
 
+  .avatarPreview {
+    border: 0;
+
+    .cropit-preview-image-container {
+      border-radius: 60px;
```

Comment:
It would be slightly safer to set the radius to 50%, that is always a circle no matter what the actual size is.